### PR TITLE
Trim language and improve info architechture

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
These edits have to central goals:
- Make the doc more readable but simplifying sentences, splitting paragraphs, et cetera
- Make the Information structure more obvious by providing hints about links, creating formatting, using top-level pages, etc

### CODE REVIEW

File: src/data/markdown/docs/07 extensions/01 Getting started/04 Create/02 Output Extensions.md [hunk 1]
```diff
@@ -13,12 +14,25 @@ Some potential reasons for a custom extension could include:
 Like [JavaScript extensions](/extensions/getting-started/create/javascript-extensions/), output extensions rely
 on the extension author to implement specific APIs.
 
-## Writing a simple extension
+## Before you start:
 
-The core of an Output extension is a struct that implements the [`output.Output`](https://pkg.go.dev/go.k6.io/k6/output#Output) 
+To run this tutorial, you'll need the following applications installed:
+- k6
```
Comment (javaducky) [inline src/data/markdown/docs/07 extensions/01 Getting started/04 Create/02 Output Extensions.md:20]: Same comment that we could exclude `k6` here.

File: src/data/markdown/docs/07 extensions/01 Getting started/04 Create/02 Output Extensions.md [hunk 2]
```diff
@@ -13,12 +14,25 @@ Some potential reasons for a custom extension could include:
...
+- Git
+
+You also need to install xk6:
+
+```
+go install go.k6.io/xk6/cmd/xk6@latest
+```
+
+## Write a simple extension
+
```
Comment (javaducky) [inline src/data/markdown/docs/07 extensions/01 Getting started/04 Create/02 Output Extensions.md:31]:
```suggestion

First, set up a directory to work in.

```bash
$ mkdir xk6-output-logger; cd xk6-output-logger; go mod init xk6-output-logger

```
```

File: src/data/markdown/docs/07 extensions/01 Guides/02 bundling.md [hunk 3]
```diff
@@ -3,32 +3,34 @@ title: 'Build a k6 binary with extensions'
...
+To use an extension that you found on the [Extension page](/extensions/getting-started/explore/) or the [xk6 GitHub topic](https://github.com/topics/xk6),
+you need to build a binary.
 
-> You'll first need to set up [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/). 
-> Make sure that your `$PATH` environment variable is updated so that `go version` returns 
-> the correct version.
+
+## Before you start
+
+- Install k6
```
Comment (javaducky) [inline src/data/markdown/docs/07 extensions/01 Guides/02 bundling.md:12]: Similar to the other comments, it is not necessary to have k6 preinstalled.

**Review by javaducky — CHANGES_REQUESTED** [review]

File: src/data/markdown/docs/07 extensions/01 Getting started/04 Create/02 Output Extensions.md (hunk 1 above)
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/07 extensions/01 Getting started/04 Create/02 Output Extensions.md:20]:
```suggestion
```

File: src/data/markdown/docs/07 extensions/01 Guides/02 bundling.md (hunk 3 above)
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/07 extensions/01 Guides/02 bundling.md:12]:
```suggestion
```

File: src/data/markdown/docs/07 extensions/01 Getting started/04 Create/01 JavaScript Extensions.md [hunk 4]
```diff
@@ -1,19 +1,43 @@
...
+* New network protocols
+* Improved performance
+* Features not supported by k6 core
 
-## Writing a simple extension
+## Before you start
 
-A simple JavaScript extension requires a struct that exposes methods to be called from a k6 test script. For example:
+To run this tutorial, you'll need the following applications installed:
+- k6
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/07 extensions/01 Getting started/04 Create/01 JavaScript Extensions.md:18]:
```suggestion
```

**Review by javaducky — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**